bigtable/client/internal/endian.h

```diff
@@ -0,0 +1,153 @@
+// Copyright 2018 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_INTERNAL_ENDIAN_H_
+#define GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_INTERNAL_ENDIAN_H_
+
+#include "bigtable/client/internal/throw_delegate.h"
+#include <bitset>
+#include <limits>
+#include <vector>
+
+#if defined(_MSC_VER)
+#include <stdlib.h>
+#else
```

Comment:
You need to check that the compiler is GCC or Clang here (look at `internal/port_platform.h`), there are other compilers out there.